src/renderer/components/dock/pod-logs.tsx

```diff
@@ -78,39 +75,81 @@ export class PodLogs extends React.Component<Props> {
     await this.load();
   }
 
+  /**
+   * Function loads more logs (usually after user scrolls to top) and sets proper
+   * scrolling position
+   * @param scrollHeight previous scrollHeight position before adding new lines
+   */
+  preload = async (scrollHeight: number) => {
+    if (podLogsStore.lines < logRange) return;
```

Comment:
The combination of these two lines is counter intuitive. Namely, if we are preloading logs I would assume that `podLogsStore.lines` to be `0`. Which is always less than `logRange`. I think that a better name than `preload` would be good.

Reply:
`preload` -> `loadMore` which is show clearer picture.